actions/contacts/unconfirm.go

```diff
@@ -41,3 +41,20 @@ func (a *Action) unconfirm(c *gin.Context) {
 	}
 	c.Status(http.StatusOK)
 }
+
+// unconfirmContact will unconfirm contact request
+// Unconfirm contact godoc
+// @Summary		Unconfirm contact
+// @Description	Unconfirm contact. For contact with status "confirmed" change status to "unconfirmed"
+// @Tags		Contacts
+// @Produce		json
+// @Param		paymail path string true "Paymail address of the contact the user wants to unconfirm"
+// @Success		200
+// @Failure		404	"Contact not found"
+// @Failure		422	"Contact status not confirmed"
+// @Failure		500	"Internal server error"
+// @Router		/v1/contacts/{paymail}/non-confirmation [PATCH]
```

Comment:
Hmm... It doesn't look the best,
maybe, if we use POST for confirmation, then we could change this to:
`DELETE /api/v1/contacts/{paymail}/confirmation`
WDYT?

Reply:
after some thought - what you proposed is better (POST "creates" a confirmation and DELETE "removes" it and `non-confirmation` seems ambigious). 
So I changed it